### PR TITLE
update: updates the landing page layout wuth new components and UI

### PR DESCRIPTION
**Pull request by chandhuDev**

### CODE REVIEW

**Comment by Vicopem01** [comment]
@chandhuDev , this pr #103,

Does it include the new landing page design?

**Comment by chandhuDev** [comment]
@Vicopem01 yes victor

**Comment by Vicopem01** [comment]
The updates are unable to build, check your updates, there's an issue causing a crash

**Comment by chandhuDev** [comment]
yup, I saw those ones, gonna push clearing fixes

**Comment by Vicopem01** [comment]
Alright, let me know when the changes are pushed, we can review again

**Comment by Vicopem01** [comment]
Also, does this update remove the waitlist field on the landing screen, if yes, please remove it and make a seperate branch and pr for that.

That seems more urgent and will have lesser issues

File: components/navigation/DesktopMenu.jsx [hunk 1]
```diff
@@ -126,21 +133,43 @@ function Dropdown({ title, dropdown, curPage }) {
...
-                );
-              })}
-            </div>
-          ))}
-        </div>
+      { dropdown.length <=1 ? 
+        <Card borderRadius="xl" fullWidth={true}>
+         <div className='w-max grid grid-cols-1 p-4 rounded-xl bg-opacity-10 bg-gradient-to-b from-[#27273A] to-black'>
+          { dropdown.map((dropItem, i) => (
+              <div className='flex flex-col gap-2'>
```
Comment (Vicopem01) [inline components/navigation/DesktopMenu.jsx:140]: add a key identifier for mapped data

Comment (chandhuDev) [reply components/navigation/DesktopMenu.jsx]: are you talking about the waitlist page button on header?

**Comment by Vicopem01** [comment]
The "Join Waitlist" has to remain on the live site.

Only the beta site has login buttons. When you push the changes, it won't bother the buttons

**Comment by chandhuDev** [comment]
I am merging this to beta  not main branch. Do you want this to be on main or beta?

**Comment by chandhuDev** [comment]
Can you also check the build logs which is related to notion for each blog

**Comment by Vicopem01** [comment]
What error is coming from notion?

**Pull request merged**